netflix_series_2019-09-23.sql

```diff
@@ -0,0 +1,45 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.27)
+# Database: netflix_series
+# Generation Time: 2019-09-23 12:26:51 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table favourites
+# ------------------------------------------------------------
+
+CREATE TABLE `favourites` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `TITLE` varchar(50) NOT NULL DEFAULT '',
+  `START DATE` date NOT NULL,
+  `END DATE` date DEFAULT NULL,
+  `NUMBER OF SEASONS` int(3) NOT NULL,
+  `NUMBER OF EPISODES` int(3) NOT NULL,
+  `DIRECTOR` varchar(30) NOT NULL DEFAULT '',
```

Comment:
Is 30 going to be enough?

Reply:
I would increase this and title to be varchar 255 in case you need it in the future